pkg/deps/config/client.go

```diff
@@ -0,0 +1,81 @@
+package config
+
+import (
+	"fmt"
+
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/types"
+	"github.com/spf13/cobra"
+
+	"github.com/pokt-network/poktroll/app/volatile"
+	"github.com/pokt-network/poktroll/pkg/client"
+	"github.com/pokt-network/poktroll/pkg/client/tx"
+)
+
+// GetTxClientGasAndFeesOptions returns a slice of TxClientOptions which
```

Comment:
Can you #PUC with a `why` explaining what happens in option 1 vs option 2 and when?